### PR TITLE
Add Chuck Code for review

### PR DESCRIPTION
**Pull request by frempongdev**
## In this PR 🔢 
I added the chuck of files from my previous To-Do-List project to be reviewed by my coding partner.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+If you like this project kindly support by suggesting changed by forking and pull requests
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank  everyone who inspired me to learn programming. Regards to the whole Microverse team, Peers, and family supporting. 
+<!-- <br>A massive Thank You to [Cindy Shin](https://www.behance.net/adagio07) who is the original designer of this [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) -->
```
Comment (chepkok3) [inline README.md]: I think this was for the event project. Kindly remove it from the README.md file.

**Review by chepkok3 — CHANGES_REQUESTED** [review]
### ToDo List Review### 
@frempongdev Well done for the good work you have done. Everything looks impressive.
so-close-so-close-seth-meyers
✔️You have followed the GitHub flow very well. 
✔️There are no linter errors in your work and the repository has been professionally documented as required. 
👏🏿 👍🏿
1. [ ] However, despite all the functionalities working well and everything looking perfect, I think it would be better to introduce a delete button that appears when you click on the ellipsis-vertical icon as indicated in the minimalist demo image: 
Screenshot 2023-01-30 at 18 40 50

> You can achieve this by dynamically creating a button element with a function that enables deleting the item when the button thrash-can icon is pressed. 
2. [x]  Generally, your project is good and meets all the requirements and quality. Great work 
image

File: README.md (hunk 1 above)
Comment (frempongdev) [reply README.md]: Alright , thank You

**Comment by frempongdev** [comment]
Thank you for pointing out the delete button issue, however, the functionality of the button is that it only appears when you are about to edit the task as shown in the demo. 

Kindly try clicking on the task field and notice that the menu button disappears and the delete button appears while the task becomes active to be edited.

**Review by chepkok3 — APPROVED** [review]
Hi @frempongdev 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 👏👏👏

**Comment by frempongdev** [comment]
Thanks @chepkok3 for approving this PR.

**Pull request merged**